sw/device/tests/spi_passthru_test.c

```diff
@@ -224,12 +225,41 @@ status_t command_processor(ujson_t *uj) {
 }
 
 bool test_main(void) {
+  if (kDeviceType == kDeviceSilicon) {
+    // On teacup board, we need to enable pull-ups on the pins connected to `WP`
+    // and `HOLD`.
+    dif_pinmux_t pinmux;
+    mmio_region_t base_addr =
+        mmio_region_from_addr(TOP_EARLGREY_PINMUX_AON_BASE_ADDR);
+    CHECK_DIF_OK(dif_pinmux_init(base_addr, &pinmux));
+
+    dif_pinmux_pad_attr_t out_attr;
+    dif_pinmux_pad_attr_t in_attr = {
+        .slew_rate = 0,
+        .drive_strength = 0,
```

Comment:
This is probably one of the reasons the clock rates have to be so slow. The drive strength and slew rate of all output buffers needs to be a reasonable value, and the minimum probably isn't for testing at speed.

Reply:
Thanks for noting that, configuring the slew rate and drive strength for the sck made it work.